### DISCUSSION TITLE
想詢問大家在桃園、中原、中壢有沒有吃過什麼隱藏版美食？

### DISCUSSION THREAD

**Discussion by 155Julia** [1101-GS4719-C/Python-Community-GS4719-C#1978] [Q&A]
哈囉各位！我最近超級好奇，有沒有人發現桃園、中原、中壢有什麼超級隱藏的美食啊？身為即將升上大三的我，已經受夠了學校周邊那些老掉牙的餐廳，真的需要一些新鮮刺激！
想像一下，你最近有沒有發現一家小店，走進去就被驚艷到了？那種感覺就像找到了寶藏一樣！🤤

當然，這邊我也整理一些自己的推薦給大家：
1. 海底撈 統領店：有人去過海底撈統領店嗎？最近我才去吃了一次，我敢打包票，這是你絕對不會後悔的選擇！而且，如果你是中央學生，還可以享受到85折優惠哦！價格實在划算，而且還能免費美甲，簡直太滿足了！
2. 榕小麵：他們的酒釀湯圓簡直是人間美味，吃了之後上癮指數直線上升，你一定要試試看！
3. 夏帕義大利麵：很讚，根本就是山下No.17
4. 九龍大湯圓：自己的隱藏版清單，冬天的時候，別忘了去九龍大湯圓暖暖身子！一碗熱騰騰的湯圓絕對是冬天最好的伴侶！
5. 十三張：口味獨特，其他店家難以復刻，推重慶小麵和紅油抄手

**Comment by EPHMR0524** [comment]
我推薦牛浜拉麵，這家拉麵店在我看到朋友的限動之前是完全不知道有這一家店的，去吃過後覺得挺好的，個人排名應該算是中壢前幾的拉麵店，但有趣的是每次我去的時候人都挺少，不知道是不是因為宣傳不夠所以沒多少人去吃，也可能是附近能選擇的食物太多了，才導致都沒什麼人（其實騎車在店家所在的那條路上根本看不到店家的招牌）

**Comment by cassy0201** [comment]
我覺得火鍋的話福柒很不錯,可以邊吃火鍋邊喝手搖,還有甜點可以吃

**Comment by hbh10601** [comment]
海底撈超級遠XDDD，我都是有回家跟朋友約出來才會吃，因為住離那邊比較近，否則要我從中央特別跑去吃真的有點太遠、耗時間。
推山下的一間生魚片丼飯，叫做鮨立，海鮮都很新鮮，還有超好喝的蝦頭鮭魚味噌湯免費續，CP值很高，有機會可以去吃吃看。

**Comment by YeeDaniel** [comment]
我推薦中原夜市的扯後腿，就如同IG上的reel拍得一樣浮誇，一整盒滿滿的雞腿排，甚至擺好滿滿3層，真的超級過癮~

**Comment by yubo0oo** [comment]
香港風味的水煎包，特別的正方形狀跟傳統台式很不一樣。共有三種口味牛肉、豬肉、蝦仁可以選擇，其中蝦仁口味是我喜歡的！以類似潤餅的外皮包進餡料精選的大隻蝦仁、高麗菜、雞蛋，放入高溫油鍋，炸到金黃酥脆。一口咬下脆口鮮甜的蝦仁搭上滑嫩的半熟蛋，爽脆多汁的高麗菜，增添多層次的口感。 
另外，牛肉口味的肉片也給得相當大方，滿滿的牛肉與食材湯汁精華合而為一，豐富的配料加上現煎的美味，怎麼可能不好吃！
地址：桃園市中壢區新生路三段828號

**Comment by yubo0oo** [comment]
香港風味的水煎包，特別的正方形狀跟傳統台式很不一樣。共有三種口味牛肉、豬肉、蝦仁可以選擇，其中蝦仁口味是我喜歡的！以類似潤餅的外皮包進餡料精選的大隻蝦仁、高麗菜、雞蛋，放入高溫油鍋，炸到金黃酥脆。一口咬下脆口鮮甜的蝦仁搭上滑嫩的半熟蛋，爽脆多汁的高麗菜，增添多層次的口感。 

另外，牛肉口味的肉片也給得相當大方，滿滿的牛肉與食材湯汁精華合而為一，豐富的配料加上現煎的美味，怎麼可能不好吃！ 
地址：桃園市中壢區新生路三段828號 

營業時間：14：00~17：30（週六日公休）

**Comment by poiu6fi** [comment]
之前家聚的時候有去山下的SOGO吃金色三麥，覺得還滿適合一群朋友去吃聚餐的，不過有點貴就是了
